pkg/server/endpoints/entryfetcher_test.go

```diff
@@ -41,12 +41,17 @@ func TestNewAuthorizedEntryFetcherWithFullCache(t *testing.T) {
 	ctx := context.Background()
 	log, _ := test.NewNullLogger()
 	clk := clock.NewMock(t)
+
 	entries := make(map[spiffeid.ID][]*types.Entry)
 	buildCache := func(context.Context) (entrycache.Cache, error) {
 		return newStaticEntryCache(entries), nil
 	}
 
-	ef, err := NewAuthorizedEntryFetcherWithFullCache(ctx, buildCache, log, clk, defaultCacheReloadInterval)
+	pruneEventsFn := func(context.Context, time.Duration) error {
```

Comment:
any of this functions are actualy using prune,
I think we'll need to add unit tests to validate prune too

Reply:
test added